Adjust smart content documentation (#588)
sulu/sulu-docs: cookbook/smart-content-data-provider.rst

         */
        private $requestStack;

        public function __construct(DataProviderRepositoryInterface $repository, ArraySerializerInterface $serializer, RequestStack $requestStack)
        {
            parent::__construct($repository, $serializer);

            $this->requestStack = $requestStack;
        }

        public function getConfiguration()
        {
            if (!$this->configuration) {
                $this->configuration = self::createConfigurationBuilder()
                    ->enableTags()
                    ->enableLimit()
                    ->enablePagination()
                    ->enablePresentAs()
                    ->enableTypes([

>                         ['type' => 'example-template-1', 'title' => 'my-translation-key-1'],

Reviewer: ```suggestion
                        ['type' => 'example-type-1', 'title' => 'my-translation-key-1'],
```

>                         ['type' => 'example-template-2', 'title' => 'my-translation-key-2'],

Reviewer: ```suggestion
                        ['type' => 'example-type-2', 'title' => 'my-translation-key-2'],
```

    structures, because it allows to filter e.g. only for pages below a certain
    page.

> There are multiple `enable...` calls, which allow you to define certain features.

Reviewer: ```suggestion
There are multiple `enable...` calls, which allow you to enable certain features in the administration interface:
```
